ibis/__init__.py

```diff
@@ -49,8 +47,17 @@
         'Number of rows to be retrieved for an unlimited table expression',
     )
 
-__version__ = get_versions()['version']
-del get_versions
+try:
```

Comment:
add a comment that this is >= PY38 so can remove in the future

Reply:
Done.